src/components/groups/ConfirmAutoFill.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+interface createProps {
+  handleAutoFill: () => void;
+  handleClose: () => void;
+}
+
+const ConfrimationCard: React.FC<createProps> = ({
+  handleAutoFill,
+  handleClose,
+}) => {
+  const handleConfirm = () => {
+    handleAutoFill();
+  };
+
+  return (
+    <div className="flex flex-col mb-4  w-[50vw] md:w-[40vw] lg:w-[30vw]">
+      <div className="info mx-auto items-center justify-content text-start">
+        <h1 className="font-bold p-2 justify-center">Are you sure?</h1>
+        <p className="text-light item-align justify-center">
+          Auto fill will fill the remaining seats in the group with earliest
+          applicants from the wait list.
+        </p>
+      </div>
+      <div className="confirmation pt-8 justify-end">
```

Comment:
`justify-end` is not applying any effect because it is not in the same class as `flex`. 
`confirmation` is this a custom class? If not there is no class called `confirmation` in tailwind, so remove it.